configs/templates/generated_interfaces_for_adapters/log/LogTemplate.gen.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package istio.mixer.adapter.log;
+
+import "mixer/v1/config/descriptor/value_type.proto";
+import "mixer/configs/mixer/TemplateExtensions.proto";
+
+option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
+option (istio.mixer.v1.config.template.template_name) = "Log";
+
+/*
+Data from Template proto
+message Constructor {
```

Comment:
How does this work? ATM the Constructor and Type are identical. What about this instead:

log.gen.proto:
```proto
// from operator's template.proto
message Type {
    map<string, istio.mixer.v1.config.descriptor.ValueType> dimensions = 1;
}

message Constructor {
    map<string, string> dimensions = 1;
}

// Needed for gRPC adapters, but we don't need to produce today.
message Instance {
    map<string, google.protobuf.Value> dimensions = 1;
}
```
log.gen.go:
```go
type Instance struct {
    Dimensions map[string]interface{}
}
```